sktime/forecasting/enbpi.py

```diff
@@ -233,21 +260,18 @@ def get_test_params(cls):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``
         """
-        from sktime.utils.dependencies import _check_soft_dependencies
-
-        deps = cls.get_class_tag("python_dependencies")
-
-        if _check_soft_dependencies(deps, severity="none"):
-            from tsbootstrap import BlockBootstrap
-
-            params = [
-                {},
-                {
-                    "forecaster": NaiveForecaster(),
-                    "bootstrap_transformer": BlockBootstrap(),
-                },
-            ]
-        else:
-            params = {}
+        params = [
+            {
+                "bootstrap_transformer": MovingBlockBootstrapTransformer(
+                    return_indices=True
+                ),
+            },
+            {
+                "forecaster": NaiveForecaster(),
+                "bootstrap_transformer": MovingBlockBootstrapTransformer(
+                    return_indices=True
+                ),
+            },
```

Comment:
let's keep the `tsbootstrap` test case too (append if dep is present)

Reply:
Done